CodeStarterCamp_Week2/Week2_Step2.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  main.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by yagom.
+//  Copyright © yagom academy. All rights reserved.
+//
+
+func generateLottoNumbers() -> Set<Int> {
+    var lottoNumbers: Set<Int> = Set<Int>()
+    
+    while lottoNumbers.count < 6 {
+        let randomNumber = Int.random(in: 1...45)
+        lottoNumbers.insert(randomNumber)
+    }
+    
+    return lottoNumbers
+}
+
+func checkLottoResult() -> Array<Int> {
+    let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+    var resultNumbers: Array<Int> = []
+
+    for number in generateLottoNumbers() {
+        if myLottoNumbers.contains(number) {
+            resultNumbers.append(number)
+        }
+    }
+    
+    return resultNumbers
+}
+
+func printLottoResult() {
+    let lottoResult = checkLottoResult()
+    
+    if lottoResult.count == 0 {
```

Comment:
배열이 비어있는지는 이렇게도 확인할 수 있으니 참고하세요 🙂
```suggestion
    if lottoResult.isEmpty == true {
```